doc/api/zlib.md

```diff
@@ -766,4 +781,5 @@ Decompress a chunk of data with [Unzip][].
 [Unzip]: #zlib_class_zlib_unzip
 [`UV_THREADPOOL_SIZE`]: cli.html#cli_uv_threadpool_size_size
 [options]: #zlib_class_options
+[`zlib.bytesWritten`]: #zlib_zlib_byteswritten
```

Comment:
A nit: it seems to be out of ASCII order (if we still follow this order in references).